doc/cordova.txt

```diff
@@ -6,6 +6,8 @@ Global Commands
     create ............................. Create a project
     help ............................... Get help for a command
     telemetry .......................... Turn telemetry collection on or off
+    config ............................. Store and retrieve environment variables 
```

Comment:
Along the same lines as my previous comment, to me, environment variables means something other than program-specifc (i.e. cordova-specific) configurations. Whatever wording we choose for the synopsis, I would also update this portion to reflect that.